### ISSUE TITLE
tests error: document is not defined

### ISSUE THREAD

**Issue by moustachiste** [bramstein/fontfaceobserver#89]
Hi, 

I have added fontfaceobserver in my codebase and now my tests are failing:
```
ReferenceError: document is not defined
    at /Users/mousta/Work/johnny/renderer/node_modules/fontfaceobserver/fontfaceobserver.standalone.js:1:21
```
We use mocha, and here is the content of the mocha.opts file:
```
--compilers js:babel-register
--require babel-polyfill
```

This is my test command:
`NODE_PATH=./src ../node_modules/.bin/mocha --opts ./mocha.opts $(find test/ -name '*.server.test.js' ! -ipath '*node_modules*')` 

I haven't found anybody else mentioning so it's probably my config, but just as a long shot if somebody knew what I can do...

thanks for your help

**Comment by bramstein** [comment]
Sorry, there was a bad release that caused this. I released a new version 2.0.7 that should fix this.

**Comment by moustachiste** [comment]
Sweet, that's fixed.

thanks
